.codecov.yml

```diff
@@ -1,10 +1,35 @@
+codecov:
+  require_ci_to_pass: no
+
 comment:
   behavior: default
```

Comment:
default behavior updates the coverage info but when updating, the creation date of the comment will stay the same. There will be an edited link with a drop-down where you can check the times updated.
there is a `new` setting which deletes old and post new one. Would considering that be better @kanchwala-yusuf ?
cc @kadel